python/torch_mlir_e2e_test/test_suite/basic.py

```diff
@@ -351,7 +351,7 @@ def forward(self, x):
 
 @register_test_case(module_factory=lambda: ConstantPad2dStaticModule())
 def ConstantPad2dStaticModule_basic(module, tu: TestUtils):
-    module.forward(tu.rand(1, 1, 20, 20) - 0.5)
+    module.forward(tu.rand(1, 1, 20, 20, low=-1))
```

Comment:
I'm aware that this replacement results in a different range. However, the only thing special about the `-0.5` is that it generated an equal number on average of negative and positive values. Using `low=-1` achieves this same goal without having to also specify `high=`, which would be the case if I had used `low=-0.5` instead.